<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsSingleBandPseudoColorRendererWidgetBase</class>
 <widget class="QWidget" name="QgsSingleBandPseudoColorRendererWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="mBandLabel">
       <property name="text">
        <string>Band</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mBandComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="mColorInterpolationLabel">
       <property name="text">
        <string>Color interpolation</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mColorInterpolationComboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="mAddEntryButton">
       <property name="text">
        <string>Add entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mDeleteEntryButton">
       <property name="text">
        <string>Delete entry</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mSortButton">
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>148</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="mLoadFromBandButton">
       <property name="toolTip">
        <string>Load color map from band</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionNewAttribute.png</normaloff>../../images/themes/default/mActionNewAttribute.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mLoadFromFileButton">
       <property name="toolTip">
        <string>Load color map from file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionFolder.png</normaloff>../../images/themes/default/mActionFolder.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mExportToFileButton">
       <property name="toolTip">
        <string>Export color map to file</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../images/themes/default/mActionFileSave.png</normaloff>../../images/themes/default/mActionFileSave.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTreeWidget" name="mColormapTreeWidget">
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Color</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Label</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="grpGenerateColorMap">
     <property name="title">
      <string>Generate new color map</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="0" column="0">
       <widget class="QLabel" name="mNumberOfEntriesLabel">
        <property name="text">
         <string>Number of entries</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="mNumberOfEntriesSpinBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="mClassificationModeLabel">
        <property name="text">
         <string>Classification mode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="mClassificationModeComboBox"/>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="mClassifyButton">
        <property name="text">
         <string>Classify</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
